spec/lrama/states_spec.rb

```diff
@@ -2096,6 +2089,25 @@ class    go to state 5
             $default  reduce using rule 4 (expr)
 
 
+        State 7
```

Comment:
I think this change is not intentional in other words no other state refers state 7 and 8. What do you think about it?